tests/e2e-vcluster/vcluster-ops/setup-vdb/base/setup-vdb.yaml

```diff
@@ -0,0 +1,41 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: vertica.com/v1beta1
+kind: VerticaDB
+metadata:
+  name: verticadb-sample
+spec:
+  image: kustomize-vertica-image
+  sshSecret: ssh-keys
+  communal:
+    includeUIDInPath: false
```

Comment:
Let's set this to true. Otherwise the test is more likely to hit communal storage is not empty when trying to create a db -- usually after a failed attempt.